lib/mix/lib/mix/tasks/help.ex

```diff
@@ -221,7 +221,20 @@ defmodule Mix.Tasks.Help do
   end
 
   defp alias_doc(task_name, note) do
-    {"Alias for " <> inspect(task_name), "mix.exs", note}
+    alias_doc = """
+    Alias for
+
+    #{format_alias(task_name)}
+    """
+
+    {alias_doc, "mix.exs", note}
+  end
+
+  defp format_alias(task) do
+    inspect(task, pretty: true, width: 0)
```

Comment:
```suggestion
    inspect(task, pretty: true, width: width() - 4)
```

Reply:
With `width() - 4` it will look like this:

![image](https://github.com/elixir-lang/elixir/assets/1561963/2a3795e2-7e54-45ef-907d-20a6aa8a6c5e)

Isn't it better to keep a single command per line?

Reply:
I see. Let's go with width: 0 then. :) Ignore this!